camelot/view/action_steps/item_view.py

```diff
@@ -85,27 +81,6 @@ def gui_run(cls, gui_context_name, serialized_step):
                     Qt.SortOrder._value2member_map_[step["order"]]
                 )
 
-
-@dataclass
-class CrudActions(DataclassSerializable):
-    """
-    A data class which contains the routes to crud actions available
-    to the gui to invoke.
-    """
-
-    admin: InitVar
```

Comment:
vermits die actions w gerund op een model context, die een link naar de admin bevat, denk ik dat die admin hier gewoon weg mag ?  tenzij de redenering is dat we de admin gaan laten bepalen wat die actions juist zijn, maar dan is het nog twijfelachtig of dat hier mt gebeuren ...